### DISCUSSION TITLE
Contributors tab not properly showing co-authored commit contributions

### DISCUSSION THREAD

**Discussion by gtwx76** [community/community#9224] [Other Feature Feedback, Questions, & Ideas]
Hello!

Co-authored commits, i.e commits attributed to more than one author using `Co-authored-by`, only attribute the contributions to the commit main author in a repository's `Contributors` tab (and not to any additional authors specified using `Co-authored-by`).

**Comment by glepnir** [comment]
I am also get this problem.

**Reply by freya274** [reply]
If the Contributors tab isn't properly showing co-authored commit contributions, it's likely because GitHub only tracks the primary author for contributions. Co-authors are recognized in the commit message but not in the contributors tab. To ensure proper attribution, make sure co-authors are listed in the commit message using the Co-authored-by tag.

**Comment by mickhence** [comment]
The Contributors tab might not properly show co-authored commit contributions due to potential issues with the repository's configuration or limitations in the platform's display functionality. Check repository settings and consider reaching out to platform support for further assistance.

**Comment by glepnir** [comment]
> `repository's configuration` 

which configuration of repo ?

**Reply by freya274** [reply]
Here are some key configurations for a repository:

Branching Strategy: Use models like Git Flow or GitHub Flow for managing development.
Default Branch: Set a default branch (usually main or develop).
Pull Requests: Enable pull requests for code reviews before merging.
Commit Messages: Follow a consistent commit message convention.
Documentation: Include a detailed README and maintain project documentation.
CI/CD Integration: Use tools like GitHub Actions for automated testing and deployment.
Issue Tracking: Utilize issue tracking for tasks and bug management.
Access Control: Set permissions to manage contributor access.
Adapt these configurations based on your team's workflow and project needs!

**Comment by ava351** [comment]
Here's the breakdown:

Git: Recognizes co-authors through "Co-authored-by" lines in the commit message.
Contributors Tab: Often only reflects the main committer (the one who pushed the commit), not additional co-authors.
This can be frustrating because it doesn't properly credit all contributors.

Here are some alternative solutions to consider:

Mention co-authors in the commit message itself: Briefly describe their contributions within the main message.
Use pull requests with clear descriptions: This allows assigning credit and outlining each contributor's role.
Consider platform-specific features: Some platforms, like GitHub, offer options to "Co-author a pull request," which can improve credit attribution.
Hopefully, these strategies can help ensure all co-authors receive proper recognition for their work!

**Comment by freya274** [comment]
If the Contributors tab isn't properly showing co-authored commit contributions, it's likely because GitHub only tracks the primary author for contributions. Co-authors are recognized in the commit message but not in the contributors tab. To ensure proper attribution, make sure co-authors are listed in the commit message using the Co-authored-by tag.
